exams/pearson/writers_test.py

```diff
@@ -186,7 +187,7 @@ def test_write_skips_invalid_rows(self):  # pylint: disable=no-self-use
 
 
 @ddt.ddt
-class CDDWriterTest(TSVWriterTestCase):
+class CDDWriterTest(TSVWriterTestCase, TestCase):
```

Comment:
I'll make those changes, but are these new testing best practices documented somewhere? Meaning, if it wasn't you reviewing this, how would this have got caught?